be/src/util/dynamic_cache.h

```diff
@@ -360,7 +377,7 @@ class DynamicCache {
         return ret;
     }
 
-    mutable std::mutex _lock;
```

Comment:
Why use bthread::Mutex?

Reply:
because the cache may be accessed in the brpc request